### PR TITLE
Process docs

### PR DESCRIPTION
**Pull request by crilleengvall**
Adds argv and platform documentation to process docs

### CODE REVIEW

File: docs/api/process.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. Darwin for MacOS, linux for Linux and win32 for Windows.
```
Comment (zeke) [inline docs/api/process.md]: Please wrap the the values in backticks, e.g. `linux`. Also, `darwin` should be lowercase.

You might also consider using an unordered list to summarize the values for each platform.

File: docs/api/process.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
...
+
+### `process.argv`
+
+An array containing the arguments passed to the application when started. To get the arguments from the renderer process use require('electron').remote.process.argv
```
Comment (zeke) [inline docs/api/process.md]: please wrap the `require...` call in backticks.

**Comment by crilleengvall** [comment]
i fixed the formatting in the latest commit

File: docs/api/process.md [hunk 3]
```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for MacOS, `linux` for Linux and `win32` for Windows.
```
Comment (zeke) [inline docs/api/process.md]: MacOS -> macOS for consistency with the rest of the docs

Sorry, I know that is very minor. Other than that, looks great!

Comment (crilleengvall) [inline docs/api/process.md]: Done!

File: docs/api/process.md [hunk 4]
```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for macOS, `linux` for Linux and `win32` for Windows.
+
+### `process.argv`
+
+An array containing the arguments passed to the application when started. To get the arguments from the renderer process use `require('electron').remote.process.argv`
```
Comment (MarshallOfSound) [inline docs/api/process.md]: In order for the docs linter to correctly understand these properties this needs to be

```markdown
A String[] array containing...
```

File: docs/api/process.md [hunk 5]
```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for macOS, `linux` for Linux and `win32` for Windows.
```
Comment (MarshallOfSound) [inline docs/api/process.md]:
```markdown
A String representing the name of the platform the app is running on.  Can be `darwin`, `linux` or `win32`.
```

Comment (MarshallOfSound) [inline docs/api/process.md]: The `can be` syntax is also so that the linter can get possible values for the String 👍

**Comment by zeke** [comment]
Thanks for your perseverance, @crilleengvall! ⭐️

**Pull request merged**